agenthub/monologue_agent/agent.py

```diff
@@ -32,7 +32,7 @@
 if config.get(ConfigType.AGENT_MEMORY_ENABLED):
     from agenthub.monologue_agent.utils.memory import LongTermMemory
 
-MAX_MONOLOGUE_LENGTH = 20000
+MAX_TOKEN_COUNT_PADDING = 512
```

Comment:
Is this roughly a similar number? 40 chars per token? That seems like a lot to me.

Reply:
Ahh nvm--I see how it's being used differently